apps/registry/public/r/styles/interlace-theme.css

```
@@ -0,0 +1,197 @@
+   * consistency between shadcn-styled and fumadocs-styled surfaces.
+   */
+  --color-fd-muted-foreground: var(--interlace-muted-foreground);
+}
+
+/* Register the shadcn-bare tokens as Tailwind v4 theme keys so utilities
+ * like `bg-card`, `text-foreground`, `border-border`, `bg-primary` are
+ * generated. `inline` preserves the `var()` reference, so the runtime
+ * `.dark { --background: ... }` overrides above flow through to every
+ * utility. Without this block, Tailwind v4 emits no `bg-card` rule
+ * regardless of what `--card` resolves to on :root. */
+@theme inline {
```

Stylelint (17.11.0)

[error] Unexpected unknown at-rule "`@theme`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)